README.md

```diff
@@ -144,6 +144,36 @@ proxy := gomitmproxy.NewProxy(gomitmproxy.Config{
 })
 ```
 
+Or implement using `OnAuthorize` handler.
+
+```go
+proxy := gomitmproxy.NewProxy(gomitmproxy.Config{
+    ListenAddr: &net.TCPAddr{
+        IP:   net.IPv4(0, 0, 0, 0),
+        Port: 8080,
+    },
+    OnAuthorize: func(session *gomitmproxy.Session) (bool, *http.Response) {
```

Comment:
I don't see any place in the code where this function is used, have you commited everything?